js/tab.js

```diff
@@ -234,3 +237,7 @@ Tab.prototype = {
 
 };
 
+Tab.showTab = function(host, chan) {
+  var tab = document.querySelector('[data-host=' + host + '][data-channel=' + chan + ']');
```

Comment:
actually, maybe keep it uri encoded, and `[data-channel=' + encodeUri(chan) + ']'`

maybe?